### ISSUE TITLE
Support for `browser.anything?(version)`?

### ISSUE THREAD

**Issue by FezVrasta** [fnando/browser#290]
## Description

I can do this:
```
browser.ie?([">8", "<10"]) 
```

But I can't do this:
```
browser.chrome?([">50", "<45"]) 
```

Why?

## How to reproduce

N/A

## What do you expect

The same API to be available across all the supported browsers.

## What happened instead

The API isn't consistent.

## Software:

- Rails version: 3.2.22
- Browser gem version: 2.3.0

## Full backtrace

```text
wrong number of arguments (given 1, expected 0)
```

**Comment by FezVrasta** [comment]
Why have you closed this issue?

**Comment by fnando** [comment]
Because it's now implemented by e808ab828cea1b72b8349985f4d608e293a4c5ca.

**Comment by FezVrasta** [comment]
Great, thank you
